README.md

```diff
@@ -17,6 +17,14 @@ Zur Hilfestellung des Projektes steht der Dozent zur verfügung:
 
 Die Lizenzrechte sind in der Datei LICENSE.txt zu lesen. Sie beinhaltet Informationen zu Nutzungsrecht, Schutz und Übertragbarkeit des Codes zu diesem Projekt.
 
+## Ausführen des Programm
+
+Das Programm kann als .jar Datei mit Java über die Command Line folgendermaßen gestartet werden:
+
+java –jar <Pfad zum Programm>
```

Comment:
Wenn du hier vier Leerzeichen vor machst, sieht es in der README schöner aus.

Reply:
Wie meinen Sie das?

Reply:
Aktuell steht hier "java –jar <Pfad zum Programm>" - wenn du daraus "    java –jar <Pfad zum Programm>" machst (man beachte die Leerzeichen) sieht das in der README nicht mehr so aus:

java –jar <Pfad zum Programm>

sondern so:

    java –jar <Pfad zum Programm>